### DISCUSSION TITLE
Adding Wordwise for languages that have no trained spacy model.

### DISCUSSION THREAD

**Discussion by Vuizur** [xxyzz/WordDumb#198] [Ideas]
I think it would be pretty nice to support creating word wise files for rarer languages that don't have a trained spacy model, but still a lot of useful Kaikki data. Personally I am interested in Czech, in general one could think of maybe all languages with wordfreq difficulty data. 

(I absolutely love this program, it really is super useful.)

**Comment by xxyzz** [comment]
Other languages could be added, but they could only use `PhraseMatcher` without pos and lemmatization, you probably have to lower the expectation of the word wise file quality. I could try to add Czech language to the Proficiency project, but I'm more interested about adding word sense disambiguation and entity linking.

**Comment by xxyzz** [comment]
Czech language is added to the master branch, please download the test version from GitHub Actions.

**Reply by Vuizur** [reply]
Amazing, it works great. Thanks a lot!
